files/bash_aliases

```diff
@@ -37,9 +41,28 @@ alias apacherestart='sudo httpd -e info -k restart'
 alias memcached-start='memcached -I2m -d start'
 alias mariastart='brew services start mysql'
 alias mariastop='brew services stop mysql'
+alias apache-logs='cd /usr/local/var/log/httpd'
+alias codecept='php ~/work/testing-toolkit/vendor/codeception/codeception/codecept'
```

Comment:
This would be better to add testing-toolkit to $PATH since there are more useful binaries there as well (and they keep adding)

Reply:
@krukru make an issue for this please, also note other stuff that could be added to $PATH

Reply:
@flackovic what do you think?

Reply:
Yeah, i was thinking about that but left it out until we make a projects auto-clone feature.
We can add it then ? I'm not sure what happens if u add not existing folder to path?

Reply:
projects do clone to the work repo

Reply:
you just need to add testing-toolkit to projects being cloned

Reply:
@flackovic you can remove this line, we will implement codecept (and whole testing toolkit) in another PR addressing #13